src/components/ChangePlan/ChangePlan.js

```diff
@@ -272,7 +274,10 @@ const ChangePlan = ({ location, dependencies: { planService, appSessionRef } })
                         key={index}
                         path={path}
                         showFeatures={isFeaturesVisible}
-                        currentPlanType={state.currentPlan.type}
+                        avaiablePlanType={planService.getPreSelectedPlanTypePerPathType(
```

Comment:
Why add a preselected type here? the issue here is that if user plan is prepaid, logically we follow that type to be selected, the only case that there is no type to be preselected would be for `free` only. For `free` we could select the first and that's it.

Reply:
@lucianaDilizio

Reply:
I did this because the planType for free depends of the path. Since we don't have prepaid plans for PLUS, I made a function to return what is the basic plan for a specific plan and path. 
This works for prepaid plans too. If I have a prepaid plan and I want to hire a plus plan, I don't have prepaid plans there so I have to preselect a monthly plan.